module/apmpgxv5/batch_tracer_test.go

```diff
@@ -0,0 +1,147 @@
+package apmpgxv5_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v5"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.elastic.co/apm/module/apmpgxv5/v2"
+	"go.elastic.co/apm/v2/apmtest"
+	"go.elastic.co/apm/v2/model"
+	"os"
+	"testing"
+)
+
+type stmt struct {
+	query  string
+	action string
+}
+
+func TestBatchTrace(t *testing.T) {
+	host := os.Getenv("PGHOST")
```

Comment:
Maybe move this check to func?

Reply:
This check is similar to another modules, so it's not necessary, I think